doc/tutorials/beginner/display_panes.md

```diff
@@ -0,0 +1,274 @@
+:::
+
+Click [this link to the Panes Section](https://panel.holoviz.org/reference/index.html#panes) of the [Component Gallery](https://panel.holoviz.org/reference/index.html). Identify the [Markdown Reference Guide](https://panel.holoviz.org/reference/panes/Markdown.html) and open it. You don't have to spend time studying the details right now.
+
+It should look like
+
+[![Panes Gallery and Markdown Reference Guide](../../_static/images/display_panes_markdown_reference.png)](https://panel.holoviz.org/reference/index.html#panes)
+
+## Display a Matplotlib plot
+
+Run the code below.
+
+```{pyodide}
+import panel as pn
+import matplotlib.pyplot as plt
+import numpy as np
+import matplotlib
+
+matplotlib.use('agg')
+
+pn.extension()
+
+def create_matplotlib_figure(figsize=(4,3)):
+       t = np.arange(0.0, 2.0, 0.01)
+       s = 1 + np.sin(2 * np.pi * t)
+
+       fig, ax = plt.subplots(figsize=figsize)
+       ax.plot(t, s)
+
+       ax.set(xlabel='time (s)', ylabel='voltage (mV)',
+              title='Voltage')
+       ax.grid()
+
+       plt.close(fig) # CLOSE THE FIGURE!
+       return fig
+
+fig = create_matplotlib_figure()
+pn.pane.Matplotlib(fig, dpi=144, tight=True).servable()
+```
+
+:::{admonition} Note
+In the example we provide the arguments `dpi` and `tight` to the `Matplotlib` pane. You can find explanations and more arguments in the [Matplotlib Reference Guide](https://panel.holoviz.org/reference/panes/Matplotlib.html).
+:::
+
+## Display a hvPlot plot
+
+Run the code below.
+
+```{pyodide}
+import hvplot.pandas
+import numpy as np
+import pandas as pd
+import panel as pn
+
+pn.extension()
+np.random.seed(1)
+
+
+idx = pd.date_range('1/1/2000', periods=1000)
+df = pd.DataFrame(np.random.randn(1000, 4), index=idx, columns=list('ABCD')).cumsum()
+fig = df.hvplot()
+
+pn.pane.HoloViews(fig, sizing_mode="stretch_width").servable()
+```
+
+:::{admonition} Note
+[hvPlot](https://hvplot.holoviz.org/user_guide/Customization.html) is the **easy to use** plotting sister of Panel. It works similarly to the the familiar [Pandas `.plot` api](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.plot.html). hvPlot is built on top of the data visualization library [HoloViews](https://holoviews.org/). hvPlot, HoloViews and Panel are all part of the [HoloViz](https://holoviz.org/) family.
+:::
+
+## Display a Plotly plot
+
+Run the code below.
+
+```{pyodide}
+import pandas as pd
+import panel as pn
+import plotly.express as px
+
+pn.extension("plotly")
+
+data = pd.DataFrame([
+    ('Monday', 7), ('Tuesday', 4), ('Wednesday', 9), ('Thursday', 4),
+    ('Friday', 4), ('Saturday', 4), ('Sunday', 4)], columns=['Day', 'Orders']
+)
+
+fig = px.line(data, x="Day", y="Orders")
+fig.update_traces(mode="lines+markers", marker=dict(size=10), line=dict(width=4))
+fig.layout.autosize = True
+
+pn.pane.Plotly(fig).servable()
+```
```

Comment:
This example is the only one so far that does not display in the docs. I probably need to add `pn.extension("plotly") at the top of the document. How do I hide the code cell?

![image](https://github.com/holoviz/panel/assets/42288570/28867305-4d4a-4ac2-af85-755ddb59eeb8)

Reply:
fixed